api/models.py

```diff
@@ -64,3 +64,18 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event (models.Model):
+
+	EVENT_STATUS = (
```

Comment:
I am not sure if this is needed here at all. This is something that can be calculated from the date.